lib/kafka_ex/protocol/fetch.ex

```diff
@@ -66,6 +67,7 @@ defmodule KafkaEx.Protocol.Fetch do
             offset: integer,
             key: binary,
             value: binary,
+            headers: list(),
```

Comment:
We should be able to be more specific here, I think?  A list of what?

Reply:
I think now is clearer